paths/paths.go

```diff
@@ -36,6 +36,11 @@ func DefaultTailscaledSocket() string {
 	if runtime.GOOS == "darwin" {
 		return "/var/run/tailscaled.socket"
 	}
+	// TODO(crawshaw): does this path change with DSM7?
+	const synologySock = "/volume1/@appstore/Tailscale/var/tailscaled.sock" // SYNOPKG_PKGDEST in scripts/installer
```

Comment:
Does this work even for side-loaded SPKs, or only via the `@appstore`?

Reply:
It does on DSM6. I suspect there will be changes coming soon for DSM7.